### DISCUSSION TITLE
Creating a custom conversion output for mp3 no longer works

### DISCUSSION THREAD

**Discussion by dfumiman** [aclap-dev/video-downloadhelper#217] [Help and Q&A]
### Your question

Hi, ever since I updated the companion app I've been unable to use custom conversion outputs for MP3. To test, i just created a duplicate of MP3 HQ and saved that exact duplicate. When trying to convert any video to the new Copy of MP3 HQ, it will only download the original format while leaving the following error:

o is undefined

moz-extension://85571de9-72d4-4d34-b73b-64946ccaae97/background/main.js:42
aT@moz-extension://85571de9-72d4-4d34-b73b-64946ccaae97/background/main.js:42:20605
_T@moz-extension://85571de9-72d4-4d34-b73b-64946ccaae97/background/main.js:46:12485

Using the normal MP3 HQ works fine. But if I want to customize the converter in any way it will break.

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-CA
license: accepted
key (only 16 first characters): 0429184954a84827
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\u",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
probably linked to #204 , thanks for the feedback
